service/static/agent.yaml.in

```diff
@@ -75,6 +75,6 @@ items:
                       name: weave-cloud
                       key: token
               args:
-              - -agent.poll-url={{.Scheme}}://{{.Hostname}}/k8s/agent.yaml
+              - -agent.poll-url={{.Scheme}}://{{.Hostname}}/k8s/agent.yaml?instanceID={{.InstanceID}}
```

Comment:
I think we need to escape {{ and }} otherwise the the first pass of templating will temper with it, `{{"{{"}}
{{"}}"}}`